src/ripple/protocol/Serializer.h

```diff
@@ -49,14 +49,12 @@ class Serializer
         mData.reserve (n);
     }
 
-    Serializer (void const* data,
-        std::size_t size)
+    Serializer (void const* data, std::size_t size)
     {
         mData.resize(size);
-        std::memcpy(mData.data(),
-            reinterpret_cast<
-                unsigned char const*>(
```

Comment:
any idea why the `reinterpret_cast` is here?...`void const*` is the documented type. Also he STBlob.h memcpy call doesn't cast, but we proabably have other inconsistencies like that.